DESIGN.md

```diff
@@ -122,6 +122,18 @@ let buf = &mut [0u8; 5];
 let result = guest_memory_mmap.write(buf, addr);
 ```
 
+One of the responsibilities of `GuestRegionMmap` is to provide an API
```

Comment:
On second thought, it's probably best to move something like this description and example to the doc comments  of the `is_hugetlbfs` method from the `GuestMemory` interface, where it gets better visibility.

Reply:
Updated.